core/src/main/java/com/squarespace/template/plugins/platform/ContentFormatters.java

```diff
@@ -467,6 +467,7 @@ public void apply(Context ctx, Arguments args, Variables variables) throws CodeE
   }
 
   public static class ImageMetaSrcSetFormatter extends BaseFormatter {
```

Comment:
Move the sizes array to be a field on the outer class `ContentFormatters` and mark it `static final`